module/exporter-shopware-6/src/Infrastructure/Connector/Shopware6Connector.php

```diff
@@ -107,4 +117,29 @@ private function resolveResponse(ResponseInterface $response): ?string
         }
         throw new \RuntimeException(sprintf('Unsupported response status "%s" ', $statusCode));
     }
+
+    private function generateRequestLog(ActionInterface $action): void
+    {
+        $this->logger->debug(
+            $action->getRequest()->getMethod().' '.$action->getRequest()->getUri()->getPath(),
```

Comment:
Wouldn't it be easier to debug once a message is common and those are context parameters as well?